Fix some download links + wording; Add manual install for sdk linux (#16014)
dotnet/docs: docs/core/install/sdk.md

In this article, you'll learn how to install the .NET Core SDK. The .NET Core SDK is used to create .NET Core apps and libraries. The .NET Core runtime is always installed with the SDK.

::: zone pivot="os-windows,os-macos"

## Install with an installer

Both Windows and macOS have standalone installers that can be used to install the .NET Core 3.0 SDK.

- Windows [x64 (64-bit) CPUs](https://dotnet.microsoft.com/download/dotnet-core/3.0) | [x86 (32-bit) CPUs](https://dotnet.microsoft.com/download/dotnet-core/3.0)
- macOS [x64 (64-bit) CPUs](https://dotnet.microsoft.com/download/dotnet-core/3.0)

::: zone-end

::: zone pivot="os-linux"

## Install with a package manager

You can install the .NET Core SDK with many of the common Linux package managers. For more information, see [Linux Package Manager - Install .NET Core](linux-package-managers.md).

## Download and manually install

> After [downloading](#all-net-core-downloads) a .NET Core binary release. Open a terminal and run the following commands to extract the SDK and make the commands available at the terminal.

Reviewer: ```suggestion
To extract the SDK and make the commands available at the terminal, first [download](#all-net-core-downloads) a .NET Core binary release. Then, open a terminal and run the following commands:
```

```bash
mkdir -p $HOME/dotnet && tar zxf dotnet-sdk-3.0.101-linux-musl-x64.tar.gz -C $HOME/dotnet
export DOTNET_ROOT=$HOME/dotnet
export PATH=$PATH:$HOME/dotnet
```

> > [!WARNING]

Reviewer: ```suggestion
> [!TIP]
```
Seems more like a tip than a warning that something bad is going to happen.

::: zone-end

> ## Install along side Visual Studio Code

Reviewer: ```suggestion
## Install alongside Visual Studio Code
```

Visual Studio Code is a powerful and lightweight source code editor that runs on your desktop. Visual Studio Code is available for Windows, macOS, and Linux.

While Visual Studio Code doesn't come with an automated .NET Core installer like Visual Studio does, adding .NET Core support is simple.

01. [Download and install Visual Studio Code](https://code.visualstudio.com/Download).

> 01. [Download and install the .NET Core SDK](https://dotnet.microsoft.com/download/dotnet-core/3.0).

Reviewer: ```suggestion
01. [Download and install the .NET Core SDK](https://dotnet.microsoft.com/download/dotnet-core).
```
For future-proofing.
